flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/common/linalg/BLAS.java

```diff
@@ -19,26 +19,53 @@
 
 package org.apache.flink.ml.common.linalg;
 
+import org.apache.flink.util.Preconditions;
+
 /**
  * A utility class that provides BLAS routines over matrices and vectors.
  */
 public class BLAS {
+
+	// For level-1 routines, we use Java implementation.
 	private static final com.github.fommil.netlib.BLAS NATIVE_BLAS = com.github.fommil.netlib.BLAS.getInstance();
+
+	// For level-2 and level-3 routines, we use the native BLAS.
 	private static final com.github.fommil.netlib.BLAS F2J_BLAS = com.github.fommil.netlib.F2jBLAS.getInstance();
```

Comment:
comment and code doesn't align. 
* the field name is `NATIVE_BLAS` for the level-1, 
* the field name is `F2J_BLAS` for level-2,3. 
any naming conversion problem?

Reply:
Thanks, we have corrected the java doc.